src/compnents/SearchForm/SearchForm.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
 
 const SearchForm = (props) => {
-  const { onClick } = props;
-
-  
-  const handleClick = (event) => {
-    event.preventDefault();
-    onClick(event);
-  };
+  const { onClick, children } = props;
 
   return (
     <form className='search-form'>
-      <input
-        type='text'
-        className='search-form__input'
-        name='search'
-        id='input-search'
-        placeholder=' Enter topic'
-        required
-      />
+      {children}
       <button type='submit' className='search-form__button' onClick={onClick}>
```

Comment:
`onClick={onClick}`

please, delete it


Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses